pkg/controller/storagecluster/reconcile.go

```diff
@@ -584,10 +584,27 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 				Spec: corev1.PodSpec{
 					RestartPolicy:      corev1.RestartPolicyNever,
 					ServiceAccountName: "rook-ceph-system",
+					Volumes: []corev1.Volume{
+						{
+							Name:         "ceph-conf-emptydir",
+							VolumeSource: corev1.VolumeSource{EmptyDir: &corev1.EmptyDirVolumeSource{}},
+						},
+					},
+
 					Containers: []corev1.Container{
 						{
 							Name:  "operator",
 							Image: os.Getenv("ROOK_CEPH_IMAGE"),
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									MountPath: "/var/lib/rook",
+									Name:      "rook-config",
```

Comment:
Should this be an env var instead of a volume like in the [rook example](https://github.com/rook/rook/blob/release-1.4/cluster/examples/kubernetes/ceph/osd-purge.yaml#L18-L48)?

Reply:
Yes. Updated